### ISSUE TITLE
Code of conduct: "don't hit on your fellow students"

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/master-reference#412]
Our code of conduct talks about ensuring *"a sexualised environment is not created."*.

I just wonder if a very short (and stronger) version of this in words that everyone can understand might be: 

> ***"For everyone's peace of mind, please do not hit on anyone while they are studying on the course"***

I am just a bit concerned that not everyone is capable of differentiating between *"welcome"* and *"unwelcome"* sexual attention (arguably, some of us are hard-wired not to understand it) and just tellling people to keep their hands off each other *during the course* will be more likely to guarantee a good experience for all than language that may be open to misinterpretation by some.

**Comment by oliverjam** [comment]
I think using plain direct language is a good idea. You don't want to dance around things just because it's slightly awkward when it comes to a code of conduct. "Don't hit on anyone" is unambiguous and can't be misinterpreted.

**Comment by ZooeyMiller** [comment]
I would totally agree with the rephrasing and think being specific is better. The current wording is ambiguous and can be kind of confusing. 

The only thing I would ask is for an extension. Currently it is 

> Do not hit on anyone while they are studying **on** the course

I would say that any arguments for saying don't hit on people on the course can be extended to "don't hit on people in the space/who work here" so would ask that the wording is altered to reflect this.

**Comment by emilyb7** [comment]
I agree with Zooey about extending this to everyone in the space. Also, I would argue that hitting on someone is not the only way that a sexualised environment can be created, so I don't think the current wording should necessarily be replaced so much as added to.

**Comment by m4v15** [comment]
Just to note, that people whose first language isn't English might not know the term "hit on", so I agree with @emilyb7 re: adding this to the current wording as opposed to replacing it.

**Comment by claireinez** [comment]
I agree that the wording should be more formal and specifically targeted at unacceptable behaviour. I don't think it is entirely helpful to plaster over the problem of certain individuals not understanding what unwanted sexual advances are by forbidding consensual relationships to form. Perhaps it would be useful to clarify that "Sexual harassment is unwanted conduct of a sexual nature whether it is verbal, non verbal or physical."

**Comment by zurda** [comment]
Is this discussion ongoing, or was this resolved?
